lib/rendered-multi-select/version.rb

```diff
@@ -1,3 +1,3 @@
 module RenderedMultiSelect
-  VERSION = "1.1.2"
+  VERSION = "1.1.3"
```

Comment:
Random: Do we want to `freeze` this?

Reply:
It looks like we're inconsistent about it, but I'll freeze this one.